src/connection/connection.js

```diff
@@ -133,21 +133,22 @@ class Connection {
 
     /**
      * User provided tags
+     * @type {string[]}
      */
     this.tags = []
   }
 
   /**
    * Get connection metadata
-   * @return {Object}
+   * @this {Connection}
    */
   get stat () {
     return this._stat
   }
 
   /**
    * Get all the streams of the muxer.
-   * @return {Array<*>}
+   * @this {Connection}
```

Comment:
Why this change?

Reply:
I've hinted that context of this is Connection so it will automatically pickup type of this._getStreams().